packages/mobile/src/components/Files/ChatFileOverview.tsx

```diff
@@ -174,8 +211,22 @@ export const ChatFileOverview = memo(
       });
       if (medias.didCancel) return;
 
+      const newAssets: ImageSource[] =
+        medias.assets?.map((value) => ({
```

Comment:
Perhaps interesting to move this code into a helper somewhere? Saw it come back back a couple times